buildSrc/src/main/groovy/org/elasticsearch/gradle/test/ClusterFormationTasks.groovy

```diff
@@ -147,6 +150,34 @@ class ClusterFormationTasks {
                     into node.baseDir
                 }
                 break;
+            case 'rpm':
+                File rpmDatabase = new File(node.baseDir, 'rpm-database')
+                File rpmExtracted = new File(node.baseDir, 'rpm-extracted')
+                extract = project.tasks.create(name: name, type: Exec, dependsOn: extractDependsOn) {
+                    commandLine 'rpm', '--badreloc', '--nodeps', '--noscripts', '--notriggers',
+                        '--dbpath', rpmDatabase,
+                        '--relocate', "/=${rpmExtracted}",
+                        '-i', project.configurations.elasticsearchDistro.singleFile
```

Comment:
We need to wrap this in a closure, so the singleFile is not called until runtime. Otherwise it will trigger a dependency resolution at configuration time. Look for other uses of singleFile in cluster formation.